### ISSUE TITLE
Support Keyboard Navigation - Workspaces

### ISSUE THREAD

**Issue by larenelg** [datalust/seq-tickets#965]
Fix keyboard navigation in workspaces, currently it's not accessible with just the keyboard.

**Comment by nblumhardt** [comment]
Done!
